mitosheet/mitosheet/api/graph/graph_utils.py

```diff
@@ -102,58 +85,101 @@ def get_graph_title(
     return (" ").join(graph_title_components)
 
 
-def get_barmode(graph_type: str) -> Optional[str]:
+def get_graph_labels(
+    x_axis_column_headers: List[ColumnHeader], y_axis_column_headers: List[ColumnHeader]
+) -> Tuple[str, str]:
     """
-    Helper function for determing the barmode to apply to the graph creation function, depending
-    on the type of graph being created
+    Helper function for determining the x and y axis titles,
+    for the scatter plot and bar chart.
     """
-    if graph_type == BOX:
-        return "stack"
-    elif graph_type == HISTOGRAM:
-        return "group"
-    elif graph_type == BAR:
-        return "group"
+    if x_axis_column_headers == [] and y_axis_column_headers == []:
```

Comment:
When checking for empty lists, please use `len(...)`- it's much more readable and commonly used.